react/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@craco/craco": "^6.4.0",
+    "@craco/craco": "^6.4.3",
```

Comment:
Why have you done version updates here?
Any issue encountered?

Reply:
Yes Sir Like yesterday we are facing an error and we try to install npm but still same error occur to i serached why the error is comming so on stakoverflow they suggest to install craco after that my npm is started

Reply:
Well, I am not convinced.
[](https://github.com/gsoft-inc/craco/compare/v6.4.0...v6.4.3)

See above compare. You will not see any difference that looks to resolve anything.

Reply:
https://github.com/gsoft-inc/craco/compare/v6.4.0...v6.4.3